containers/azure-functions-pwsh-7/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,6 @@
+# Find the Dockerfile for mcr.microsoft.com/azure-functions/powershell:3.0-powershell7-core-tools at this URL
```

Comment:
Replace the 7 with ${VARIANT}

Also don't forget to change the file name now

Reply:
Done